nilearn/maskers/tests/test_surface_labels_masker.py

```diff
@@ -150,7 +150,7 @@ def test_surface_label_masker_check_output(surf_mesh, rng):
     surf_img = SurfaceImage(surf_mesh(), data)
     signal = masker.transform(surf_img)
 
-    assert signal.shape == (surf_img.shape[1], masker.n_elements_)
+    assert signal.shape == (1, masker.n_elements_)
```

Comment:
Wondering whether the output signal being 2D after masking is consistent with what we do with volume images? Only relevant when the input has one time point.

Would always be 2D otherwise.

Reply:
not sure but I would say that we can be flexible in what we accept as input but we should be predictible in what give as output: so I don't think the shape of the output should depend on the shape of the input.